keyboards/chocv/keymaps/default/readme.md

```diff
@@ -0,0 +1,15 @@
+# The default keymap for ChocV is Manna Harbour Miryoku
+
+For details and building instructions and options please see the
+[The official Miryoku QMK
+Repo](https://github.com/manna-harbour/miryoku_qmk/tree/miryoku/users/manna-harbour_miryoku)
+
+But the tl;dr version is
+
+make chocv:manna-habour_miryoku
```

Comment:
```suggestion
make chocv:default
```

Reply:
Totally willing to change this example as well as the main readme.md , but the Miryoku keymap will be available at build time by virtue of the community layout.  Are you okay if we leave this one as is?

If not, no problem, I'll change it!

Reply:
You can include both `make` commands. "Run this make command to compile the very basic test firmware : xxx. Run this make command to compile the 'real' default keymap: yyy"

Reply:
That's a great idea, I've made the keymap readme more informative and included both make commands.  Thank you @precondition !